hack/util.sh

```diff
@@ -641,6 +641,28 @@ function wait_for_registry {
 }
 
 
+function install_heapster {
+	echo "[INFO] Installing Heapster"
+
+	# set up the deployer service account
+	echo '{"kind":"ServiceAccount","apiVersion":"v1","metadata":{"name":"metrics-deployer"},"secrets":[{"name": "metrics-deployer"}]}' | oc create -f - --config="${ADMIN_KUBECONFIG}" -n openshift-infra
```

Comment:
Ah, AFAICT, `oc create serviceaccount` doesn't allow you to manually specify secret names.  That's probably why.